provider/secret_descriptor.go

```diff
@@ -145,11 +163,17 @@ func (p *SecretDescriptor) GetSecretType() (stype SecretType) {
 
 // Return a descriptor for a jmes object entry within the secret
 func (p *SecretDescriptor) getJmesEntrySecretDescriptor(j *JMESPathEntry) (d SecretDescriptor) {
+	permission := j.FilePermission
+	if permission == "" {
```

Comment:
Nit: This guard clause is different from the ones on L210 and L222. Is there a reason for that? No need to fix, just making sure I'm not missing anything.